code/modules/uplink/uplink_items.dm

```diff
@@ -75,6 +75,7 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	var/player_minimum //The minimum crew size needed for this item to be added to uplinks.
 	var/purchase_log_vis = TRUE // Visible in the purchase log?
 	var/restricted = FALSE // Adds restrictions for VR/Events
+	var/list/restricted_species = list() //Limits items to a specific species. Hopefully.
```

Comment:
It should be null by default, we don't need empty lists on everything

Reply:
In this case no and that's why I put this review comment here. Those other entries are in the wrong here and it's not worth wasting memory to be consistent when it's conveying no information.